### PR TITLE
Add a metadata header to hazard curves and maps in CSV format

### PR DESCRIPTION
**Pull request by micheles**
As requested by Graeme. Tests and demos are running here: https://ci.openquake.org/job/zdevel_oq-engine/2070/

### CODE REVIEW

**Comment by g-weatherill** [comment]
Can you add the header lines into the csv files for the mean and quantile results (for maps, curves and UHS)?

File: openquake/commonlib/export/hazard.py [hunk 1]
```diff
@@ -265,28 +265,29 @@ def add_imt(fname, imt):
...
-def export_hcurves_by_imt_csv(key, dest, sitecol, curves_by_imt,
-                              imtls, investigation_time=None):
+def export_hcurves_by_imt_csv(key, kind, rlzs_assoc, fname, sitecol,
+                              curves_by_imt, oq):
     """
     Export the curves of the given realization into CSV.
 
     :param key: output_type and export_type
     :param dest: name of the exported file
     :param sitecol: site collection
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Please add to the docstring also the parameters `kind`, `rlzs_assoc` and `fname` and remove `dest`

File: openquake/commonlib/export/hazard.py [hunk 2]
```diff
@@ -314,6 +314,17 @@ def hazard_curve_name(dstore, ekey, kind, rlzs_assoc):
     return fname
 
 
+def _comment(rlzs_assoc, kind, investigation_time):
+    rlz = rlzs_assoc.get_rlz(kind)
+    if not rlz:
+        return '%s, investigation_time=%s' % (kind, investigation_time)
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Please remove the space after comma here, or add spaces also below

**Comment by ptormene** [comment]
Apart from minor comments, LGTM

**Pull request merged**